.gitignore

```diff
@@ -23,8 +23,10 @@
 # Autogenerated Jetbrains Rider plugin
 /[Aa]ssets/Plugins/Editor/JetBrains*
 
-# Visual Studio cache directory
+# Visual Studio generated files
 .vs/
+.vsconfig
+.vscode/
```

Comment:
Is this necessary? As `.vscode` is within the `/[Ll]ibrary/` part of the gitignore already

Reply:
i made a mistake, please ignore my previous comment